src/EditorFeatures/Test2/IntelliSense/SignatureHelpControllerTests.vb

```diff
@@ -64,7 +64,6 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             Dim handled = controller.TryHandleUpKey()
 
             Assert.False(handled)
-            GetMocks(controller).PresenterSession.Verify(Sub(p) p.Dismiss(), Times.Once)
```

Comment:
❔ Does it still dismiss if the caret moves to a location which is not inside the argument list? What if it moves to a location which is inside an argument list, but is not inside the _same_ argument list?

Reply:
Good call. I assumed that the caret position changed event handler would take care of it, like it does when clicking in a different argument list, but it turns out mouse clicks dismiss the tooltip before that event fires.

That means with this change, navigating to another argument list with the keyboard keeps signature help open, and updates it to the new argument list. I'm not quite sure its a good feature or not, so for now I'll update this to keep the existing functionality, and dismiss.

Reply:
Updated so caret moves outside of the current models span will dismiss sig help.